.gitignore

```diff
@@ -11,3 +11,4 @@
 /in-market-test/hpke/bin
 /in-market-test/hpke/lib
 /in-market-test/hpke/pyvenv.cfg
+input-data-*.txt
```

Comment:
is it related to this change?

Reply:
No. Just a minor measure to avoid pushing those input files to the repo by mistake.